code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -632,13 +632,12 @@
 		return //won't work if dead
 	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Problems?", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow")
 	var/emote = input("Please, select a status!", "AI Status", null, null) in ai_emotions
-	for (var/obj/machinery/M in GLOB.machines) //change status
+	for (var/obj/machinery/M in GLOB.ai_status_displays) //change status of displays

```

Comment:
Remove the machinery typecast here, it's not being used for anything besides an extra typecheck.